web_console/pages/admin/federation.jsx

```diff
@@ -127,10 +298,40 @@ export default function FederationList() {
     setFields(mapValueToFields(federation, fields));
     setFormVisible(true);
   };
+
+  const checkValue = (key, value) => {
+    if (key === 'Env' && value) {
+      for (let {name, value} of value) {
+        if (name === '' || value === '') {
```

Comment:
the `env` value could be `valueFrom` also

Reply:
what does it means? here is checking if value is empty

Reply:
there are some prerequisites of Kubernetes. `env` is the configuration of container, u could configure its values through `value` and `valueFrom` (which means a reference, checkout official [wiki](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-pod-fields-as-values-for-environment-variables) for more detail) :)